modules/backend/lang/en/lang.php

```diff
@@ -141,7 +141,7 @@
             'name_field' => 'Name',
             'description_field' => 'Description',
             'is_new_user_default_field_label' => 'Default group',
-            'is_new_user_default_field_comment' => 'Add new administrators to this group by default',
+            'is_new_user_default_field_comment' => 'Add new administrators to this group by default.',
```

Comment:
I know that such changes are insignificant. I will try not to bother you again for these.

I am actually translating in Turkish. Current Turkish is very bad, and could not catch the standards. At this moment I encounter small things in English files. 

For now I only see missing single quotes on variables like :id, :class, :model, some have some not. It's okay if you say "stay like this".

Another thing is on/off buttons, my Turkish on/off is a little more lengthy, I modify css with plugin for on/off button width.

And for time format, why we not have language variables for that. We can define it in own yaml, but I think we need a language variable for system wide. I know some time variable is come from moment.js language.